dali/operators/imgcodec/peek_image_shape.cc

```diff
@@ -53,6 +51,20 @@ ImgcodecPeekImageShape::ImgcodecPeekImageShape(const OpSpec &spec)
   }
   use_orientation_ = spec.GetArgument<bool>("adjust_orientation");
   image_type_ = spec.GetArgument<DALIImageType>("image_type");
+
+  nvimgcodecInstanceCreateInfo_t instance_create_info;
```

Comment:
```suggestion
  nvimgcodecInstanceCreateInfo_t instance_create_info = {};
```
Otherwise we might miss new members.